src/output/outputMod.F90

```diff
@@ -1504,7 +1511,25 @@ subroutine storeSurfsolInBuffer(sps, buffer, nn, blockID,   &
        rangeFace(2,1:2) = rangeCell(3,1:2)
        iiMax = jl; jjMax = kl
 
-       ww1    => w(1,1:,1:,:);   ww2    => w(2,1:,1:,:)
+       ! have to get the mesh coordinates further down in order to compute
```

Comment:
i feel the documentation is a bit messy here. It is not a huge concern, but we may want to clean things up a bit... @marcomangano

Reply:
@SichengHe let me know if it looks better now